app/controllers/concerns/page_tool_tip.rb

```diff
@@ -0,0 +1,22 @@
+module PageToolTip
```

Comment:
And you can take good thoughts from this [stackoverflow question](http://stackoverflow.com/questions/14541823/how-to-use-concerns-in-rails-4) about Concerns concept!